src/JustEat.StatsD/UdpTransport.cs

```diff
@@ -44,6 +45,21 @@ public void Send(string metric)
             }
         }
 
+        public void Send(ArraySegment<byte> metric)
```

Comment:
Null check?

Reply:
ArraySegmentis a struct and could be constructed with the default constructor, resulting in the null array. I will add a null check to the Array field.